### ISSUE TITLE
请求贡献项目的模板|Request for a Template to Contribute to the Project

### ISSUE THREAD

**Issue by Exorcism0666** [SpecterShell/Dumplings#2]
你好，SpecterShell，
----------
我对你的Dumplings项目印象深刻，非常欣赏你所做的工作。我最近fork了你的项目，我想在我这边用我最喜欢的软件做同样的事情。

然而，我遇到了一些困难，因为我没有用PowerShell编程来完成这个特定任务的模板。你能否提供一个模板或者指导我该如何进行，特别是对于在GitHub上发布的软件？

我认为这也可能帮助对你的项目感兴趣的其他贡献者。

非常感谢你抽出时间，以及你在Dumplings上所做的出色工作。

顺祝商祺，
Exorcism

Howdy SpecterShell,
----------
I am very impressed with your Dumplings project and greatly appreciate the work you have done. I recently forked your project and would like to do the same on my end with my favorite software.

However, I am encountering difficulties as I do not have a template for programming in PowerShell for this specific task. Would it be possible for you to provide a template or guidelines on how to proceed, especially for software hosted on GitHub in the release category?

I think this could also help other contributors who are interested in your project.

Thank you very much for your time and for the incredible work you have done on Dumplings.

Best regards,
Exorcism

**Comment by SpecterShell** [comment]
If you want to automate for your project, you can take a look at this project instead: https://github.com/vedantmgoyal2009/winget-releaser

**Comment by Exorcism0666** [comment]
I had looked more closely before but I suppose you have to be the "owner" of a repository to be able to automate the process?

**Comment by Exorcism0666** [comment]
I had looked more closely before but I suppose you have to be the "owner" of a repository to be able to automate the process?

**Comment by SpecterShell** [comment]
Yeah, only people who have write access to the projects can add this GitHub Action to them, and this project is for those I don't have write access to.

This project is still working in progress. I will add some docs and templates once it is stable enough. Sorry for any inconvenience.

**Comment by Exorcism0666** [comment]
No worries, I look forward to receiving your documentation!

**Comment by Exorcism0666** [comment]
@SpecterShell it's quite special and I don't know why it doesn't check if there is an update to do when I add manifests. https://github.com/Exorcism0666/Dumplings/actions/runs/8271009457/job/22629809319

**Comment by SpecterShell** [comment]
> @SpecterShell it's quite special and I don't know why it doesn't check if there is an update to do when I add manifests. [Exorcism0666/Dumplings/actions/runs/8271009457/job/22629809319](https://github.com/Exorcism0666/Dumplings/actions/runs/8271009457/job/22629809319)

Yeah it does not rely on winget-pkgs when checking the updates, so it won't do anything for new manifests but writing what it gets to a file in the repo during the first run, which will be compared with during the subsequent runs.

**Comment by Exorcism0666** [comment]
> > @SpecterShell it's quite special and I don't know why it doesn't check if there is an update to do when I add manifests. [Exorcism0666/Dumplings/actions/runs/8271009457/job/22629809319](https://github.com/Exorcism0666/Dumplings/actions/runs/8271009457/job/22629809319)
> 
> Yeah it does not rely on winget-pkgs when checking the updates, so it won't do anything for new manifests but writing what it gets to a file in the repo during the first run, which will be compared with during the subsequent runs.

Oooooo okay, I take note, thank you!

**Comment by Dragon1573** [comment]
> [!TIP]
> 
> @Exorcism0666, 👋🏼 
> 
> I've asked for providing a template for GitHub Releases. Currently you can use these packages as template.
> 
> - Related #39

**Comment by SpecterShell** [comment]
Right now, you can use the agent skills in https://github.com/SpecterShell/Dumplings/tree/main/.agents/skills for authoring new package manifests and tasks.
